src/quo/components/links/internal_link_card/view.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo.components.links.internal-link-card.view
+  (:require [quo.components.links.internal-link-card.channel.view :as channel.view]
+            [quo.components.links.internal-link-card.community.view :as community.view]
+            [quo.components.links.internal-link-card.user.view :as user.view]))
+
+(defn view
```

Comment:
Consider adding doc string, or even better we can start using Malli Schama

Reply:
Done